lib/src/reporter.dart

```diff
@@ -93,7 +97,11 @@ class Reporter {
       color && message.isNotEmpty ? pen(message) : message;
 
   void _log(IOSink sink, String message, {bool shout: false}) {
-    if (quiet && !shout) return;
+    // Ensure that even if quiet or shout are null, the conditional
+    // will result in a boolean
+    if (quiet == true && shout != true) {
```

Comment:
I don't think we should have to worry about this case; these should never be null unless someone explicitly sets them to null, in which case the program should probably fail.